src/interfaces/Product.ts

```diff
@@ -0,0 +1,18 @@
+interface Product {
+    id: number;
+    title: string;
+    price: number;
+    description: string;
+    images: string[];
+    creationAt: Date;
+    updatedAt: Date;
+    category: {
```

Comment:
use Category interface instead of duplication same code
```suggestion
    category: Category
```

Reply:
Fixed